### ISSUE TITLE
log into a file

### ISSUE THREAD

**Issue by iav-DaBi** [iavofficial/IAV_Distortion#14]
Logging into a log file
- The path of the log file need to be configurable
- The size of the log file need to be limited
- Every log entry must receive a timestamp

**Comment by iav-BjRo** [comment]
basic logger configuration could look something like:
```
    logger = logging.getLogger(__name__)
    logging.basicConfig(filename='logfile_IAV-Distortion.log',
                        encoding='utf-8',
                        level=logging.DEBUG,
                        format='%(asctime)s - %(levelname)s: %(message)s')

    handler = RotatingFileHandler('logfile_IAV-Distortion.log', maxBytes=20000, backupCount=5)
    formatter = logging.Formatter('%(asctime)s - %(levelname)s: %(message)s')
    handler.setFormatter(formatter)
    logger.addHandler(handler)
    
    logging.info("-------------------------")
    logging.info(" STARTING IAV-Distortion ")
    logging.info("-------------------------")
```

**Comment by iav-BjRo** [comment]
not completed in Release_v2.1.0 -> has to be planed -> set to 'no milestone'

**Comment by m-braunschweig** [comment]
Merged in #129
